src/repart/repart.c

```diff
@@ -8780,6 +8836,11 @@ static int parse_argv(int argc, char *argv[], X509 **ret_certificate, EVP_PKEY *
                         return log_error_errno(r, "Failed to load private key from %s: %m", arg_private_key);
         }
 
+        if (arg_append_fstab) {
+                if (!arg_generate_fstab)
```

Comment:
merge the two if expressions into one, with &&

Reply:
Ouch. Done. Added = suffix into the message too.